extraTs/utils/helpers.ts

```diff
@@ -1,10 +1,11 @@
-export default function validateEmail(email : string) : boolean{
-    const regex = /\w+.\w+@successive.tech$/i;
-    if(regex.test(email))
-    {
-        return true;
-    }
-    else{
-        return false;
-    }
+
+export default function validateEmail(email){
+    const regex = /\w+.\w+@successive.tech$/i;
+    if(regex.test(email))
+    {
+        return true;
+    }
+    else{
+        return false;
```

Comment:
use es6 and define types

Reply:
Used ES6 and Defined Types in this file.